django_extensions/management/commands/show_template_paths.py

```diff
@@ -0,0 +1,24 @@
+import datetime
+from pathlib import Path
+
+from django.template.engine import Engine
+from django.core.management.base import BaseCommand
+
+
+class Command(BaseCommand):
+    help = "List template search paths."
+
+    def handle(self, *args, **options):
+
+        test_template_name = str(datetime.datetime.now().timestamp())
```

Comment:
Any specific reason to use the timestamp here ?

Instead of something like
```
test_template_name = "this_template_name_does_not_exist"
```

Reply:
While true, both are highly unlikely... and the benefit of a fixed string is that it would be deterministic...